packages/gcf-utils/src/logging/gcf-logger.ts

```diff
@@ -232,3 +232,18 @@ function pinoLevelToCloudLoggingSeverity(
   const UNKNOWN_SEVERITY = 'DEFAULT';
   return {severity: severityMap[level] || UNKNOWN_SEVERITY, level: level};
 }
+
```

Comment:
Could you add a comment to the GCFLogger class definition explaining why `console.log()` is insufficient?

Reply:
Done